Multisig/Cross-layer/App.swift

```diff
@@ -53,5 +51,23 @@ class App {
 
     let firebaseConfig = FirebaseConfig()
 
-    private init() { }
+    private init() {
+        let updateNotifications: [NSNotification.Name] = [
+            .selectedSafeChanged, .selectedSafeUpdated, .networkInfoChanged
+        ]
+
+        for name in updateNotifications {
+            NotificationCenter.default.addObserver(self,
+                                           selector: #selector(bindNetworkInfo),
+                                           name: name,
+                                           object: nil)
+        }
+    }
+
+    @objc func bindNetworkInfo() {
```

Comment:
could be private

Reply:
No, it's used at AppDelegate

Reply:
or can't because you use it in AppDelegate?